ocp_resources/storage_class.py

```diff
@@ -45,3 +45,57 @@ class Annotations:
         IS_DEFAULT_CLASS = (
             f"{Resource.ApiGroup.STORAGECLASS_KUBERNETES_IO}/is-default-class"
         )
+
+    class ReclaimPolicy:
+        DELETE = "Delete"
+        RETAIN = "Retain"
+
+    def __init__(
+        self,
+        name=None,
```

Comment:
Please leave here only what this specific resource needs (like reclaim_policy, provisioner , etc)
all other will be passed with kwargs

Reply:
ack